scripts/abi_check.py

```diff
@@ -120,16 +123,30 @@ def update_git_submodules(self, git_worktree_path, crypto_rev):
             raise Exception("git submodule update failed, aborting")
         if (os.path.exists(os.path.join(git_worktree_path, "crypto"))
                 and crypto_rev):
-            checkout_process = subprocess.Popen(
-                [self.git_command, "checkout", crypto_rev],
-                cwd=os.path.join(git_worktree_path, "crypto"),
-                stdout=subprocess.PIPE,
-                stderr=subprocess.STDOUT
-            )
-            checkout_output, _ = checkout_process.communicate()
-            self.log.info(checkout_output.decode("utf-8"))
-            if checkout_process.returncode != 0:
-                raise Exception("git checkout failed, aborting")
+            if crypto_repo:
+                shutil.rmtree(os.path.join(git_worktree_path, "crypto"))
```

Comment:
Probably don't want to do this except as part of a temporary work tree. It'd be a bummer to remove someones work in the crypto submodule with this. Do we always work with a temporary work tree?

Reply:
Yes, it always creates a temporary worktree. Even if it wouldn't get removed here, it would get removed in the `cleanup_worktree` function.

Reply:
OK